mlflow/xgboost.py

```diff
@@ -64,17 +68,18 @@
 _logger = logging.getLogger(__name__)
 
 
+def _get_default_pip_deps():
+    import xgboost as xgb
+
+    return ["xgboost=={}".format(xgb.__version__)]
```

Comment:
Defined this function to make the code & logic a bit cleaner.

Reply:
We can reuse this function in #https://github.com/mlflow/mlflow/pull/4518 to obtain the pip dependencies we fallback to when the dependency inference fails.

Reply:
Cool!

Reply:
I'll define the same function in othere flavors.